packages/generator/src/generator.ts

```diff
@@ -234,7 +234,9 @@ export async function generateSourcesForService(
   });
 
   service.enumTypes.forEach(enumType => {
-    logger.info(`[${service.originalFileName}] Generating enum type ...`);
+    logger.info(
+      `[${service.originalFileName}] Generating enum type ${enumType.originalName} ...`
```

Comment:
I would always put names into quotes, but I don't know whether this makes it inconsistent with other places where we log this information (same below). You decide ;)
```suggestion
      `[${service.originalFileName}] Generating enum type '${enumType.originalName}' ...`
```